### PR TITLE
redeclipse: fix licenses

### PR DESCRIPTION
**Pull request by jasonliu--**
#### Description

The source code of Red Eclipse is licensed zlib and the game data is licensed CC, so this commit splits up the `license` line so that they apply to the individual subports.

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.11.6 15G22010 x86_64
Xcode 8.2.1 8C1002

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: games/redeclipse/Portfile [hunk 1]
```diff
@@ -30,6 +29,8 @@ if {$subport eq ${name}} {
     PortGroup       app 1.0
 
     github.setup    ${name} base ${version} v
+    revision        1
```
Comment (ryandesign) [inline games/redeclipse/Portfile:32]: Why does the revision need to be increased?

Comment (jasonliu--) [reply games/redeclipse/Portfile:32]: The previous revision successfully built on the builders, so I was under the impression that they wouldn't run through the build and upload the binaries to packages.macports.org unless I revbumped. Is that not the case? If it's not, I can remove the revbump.

Comment (ryandesign) [reply games/redeclipse/Portfile:32]: That's not the case. All commits notify the builders, and they decide if a build is necessary. A change in distributability due to a change of license does trigger a build.

Comment (jasonliu--) [reply games/redeclipse/Portfile:32]: Oh, I didn't know that. Ok, I'll remove the revbump, then.

Comment (ryandesign) [reply games/redeclipse/Portfile]: See the documentation for `revision` at https://guide.macports.org/#reference.keywords.

**Pull request merged**